// @workday/canvas-tokens-web/dist/common-js/base/index.js
/**
 * Do not edit directly
 * Generated on Wed, 25 Mar 2026 17:29:04 GMT
 */

"use strict";
Object.defineProperty(exports, "__esModule", { value: true });

exports.fontSize = "--cnvs-base-font-size";
exports.baseline = "--cnvs-base-baseline";
exports.duration50 = "--cnvs-base-duration-50";
exports.duration100 = "--cnvs-base-duration-100";
exports.duration150 = "--cnvs-base-duration-150";
exports.duration200 = "--cnvs-base-duration-200";
exports.duration250 = "--cnvs-base-duration-250";
exports.duration300 = "--cnvs-base-duration-300";
exports.duration350 = "--cnvs-base-duration-350";
exports.duration400 = "--cnvs-base-duration-400";
exports.duration450 = "--cnvs-base-duration-450";
exports.duration500 = "--cnvs-base-duration-500";
exports.duration550 = "--cnvs-base-duration-550";
exports.duration600 = "--cnvs-base-duration-600";
exports.duration650 = "--cnvs-base-duration-650";
exports.duration700 = "--cnvs-base-duration-700";
exports.duration750 = "--cnvs-base-duration-750";
exports.duration800 = "--cnvs-base-duration-800";
exports.duration850 = "--cnvs-base-duration-850";
exports.duration900 = "--cnvs-base-duration-900";
exports.duration950 = "--cnvs-base-duration-950";
exports.duration1000 = "--cnvs-base-duration-1000";
exports.easingA100 = "--cnvs-base-easing-a-100";
exports.easingA200 = "--cnvs-base-easing-a-200";
exports.easingA300 = "--cnvs-base-easing-a-300";
exports.easingB100 = "--cnvs-base-easing-b-100";
exports.easingB200 = "--cnvs-base-easing-b-200";
exports.easingB300 = "--cnvs-base-easing-b-300";
exports.opacity0 = "--cnvs-base-opacity-0";
exports.opacity80 = "--cnvs-base-opacity-80";
exports.opacity100 = "--cnvs-base-opacity-100";
exports.opacity120 = "--cnvs-base-opacity-120";
exports.opacity180 = "--cnvs-base-opacity-180";
exports.opacity200 = "--cnvs-base-opacity-200";
exports.opacity250 = "--cnvs-base-opacity-250";
exports.opacity300 = "--cnvs-base-opacity-300";
exports.opacity360 = "--cnvs-base-opacity-360";
exports.opacity400 = "--cnvs-base-opacity-400";
exports.opacity500 = "--cnvs-base-opacity-500";
exports.opacity640 = "--cnvs-base-opacity-640";
exports.opacity840 = "--cnvs-base-opacity-840";
exports.amber25 = "--cnvs-base-palette-amber-25";
exports.amber50 = "--cnvs-base-palette-amber-50";
exports.amber100 = "--cnvs-base-palette-amber-100";
exports.amber200 = "--cnvs-base-palette-amber-200";
exports.amber300 = "--cnvs-base-palette-amber-300";
exports.amber400 = "--cnvs-base-palette-amber-400";
exports.amber500 = "--cnvs-base-palette-amber-500";
exports.amber600 = "--cnvs-base-palette-amber-600";
exports.amber700 = "--cnvs-base-palette-amber-700";
exports.amber800 = "--cnvs-base-palette-amber-800";
exports.amber900 = "--cnvs-base-palette-amber-900";
exports.amber950 = "--cnvs-base-palette-amber-950";
exports.amber975 = "--cnvs-base-palette-amber-975";
exports.amberA25 = "--cnvs-base-palette-amber-a25";
exports.amberA50 = "--cnvs-base-palette-amber-a50";
exports.amberA100 = "--cnvs-base-palette-amber-a100";
exports.amberA200 = "--cnvs-base-palette-amber-a200";
exports.azure25 = "--cnvs-base-palette-azure-25";
exports.azure50 = "--cnvs-base-palette-azure-50";
exports.azure100 = "--cnvs-base-palette-azure-100";
exports.azure200 = "--cnvs-base-palette-azure-200";
exports.azure300 = "--cnvs-base-palette-azure-300";
exports.azure400 = "--cnvs-base-palette-azure-400";
exports.azure500 = "--cnvs-base-palette-azure-500";
exports.azure600 = "--cnvs-base-palette-azure-600";
exports.azure700 = "--cnvs-base-palette-azure-700";
exports.azure800 = "--cnvs-base-palette-azure-800";
exports.azure900 = "--cnvs-base-palette-azure-900";
exports.azure950 = "--cnvs-base-palette-azure-950";
exports.azure975 = "--cnvs-base-palette-azure-975";
exports.azureA25 = "--cnvs-base-palette-azure-a25";
exports.azureA50 = "--cnvs-base-palette-azure-a50";
exports.azureA100 = "--cnvs-base-palette-azure-a100";
exports.azureA200 = "--cnvs-base-palette-azure-a200";
exports.blue25 = "--cnvs-base-palette-blue-25";
exports.blue50 = "--cnvs-base-palette-blue-50";
exports.blue100 = "--cnvs-base-palette-blue-100";
exports.blue200 = "--cnvs-base-palette-blue-200";
exports.blue300 = "--cnvs-base-palette-blue-300";
exports.blue400 = "--cnvs-base-palette-blue-400";
exports.blue500 = "--cnvs-base-palette-blue-500";
exports.blue600 = "--cnvs-base-palette-blue-600";
exports.blue700 = "--cnvs-base-palette-blue-700";
exports.blue800 = "--cnvs-base-palette-blue-800";
exports.blue900 = "--cnvs-base-palette-blue-900";
exports.blue950 = "--cnvs-base-palette-blue-950";
exports.blue975 = "--cnvs-base-palette-blue-975";
exports.blueA25 = "--cnvs-base-palette-blue-a25";
exports.blueA50 = "--cnvs-base-palette-blue-a50";
exports.blueA100 = "--cnvs-base-palette-blue-a100";
exports.blueA200 = "--cnvs-base-palette-blue-a200";
exports.coral25 = "--cnvs-base-palette-coral-25";
exports.coral50 = "--cnvs-base-palette-coral-50";
exports.coral100 = "--cnvs-base-palette-coral-100";
exports.coral200 = "--cnvs-base-palette-coral-200";
exports.coral300 = "--cnvs-base-palette-coral-300";
exports.coral400 = "--cnvs-base-palette-coral-400";
exports.coral500 = "--cnvs-base-palette-coral-500";
exports.coral600 = "--cnvs-base-palette-coral-600";
exports.coral700 = "--cnvs-base-palette-coral-700";
exports.coral800 = "--cnvs-base-palette-coral-800";
exports.coral900 = "--cnvs-base-palette-coral-900";
exports.coral950 = "--cnvs-base-palette-coral-950";
exports.coral975 = "--cnvs-base-palette-coral-975";
exports.coralA25 = "--cnvs-base-palette-coral-a25";
exports.coralA50 = "--cnvs-base-palette-coral-a50";
exports.coralA100 = "--cnvs-base-palette-coral-a100";
exports.coralA200 = "--cnvs-base-palette-coral-a200";
exports.green25 = "--cnvs-base-palette-green-25";
exports.green50 = "--cnvs-base-palette-green-50";
exports.green100 = "--cnvs-base-palette-green-100";
exports.green200 = "--cnvs-base-palette-green-200";
exports.green300 = "--cnvs-base-palette-green-300";
exports.green400 = "--cnvs-base-palette-green-400";
exports.green500 = "--cnvs-base-palette-green-500";
exports.green600 = "--cnvs-base-palette-green-600";
exports.green700 = "--cnvs-base-palette-green-700";
exports.green800 = "--cnvs-base-palette-green-800";
exports.green900 = "--cnvs-base-palette-green-900";
exports.green950 = "--cnvs-base-palette-green-950";
exports.green975 = "--cnvs-base-palette-green-975";
exports.greenA25 = "--cnvs-base-palette-green-a25";
exports.greenA50 = "--cnvs-base-palette-green-a50";
exports.greenA100 = "--cnvs-base-palette-green-a100";
exports.greenA200 = "--cnvs-base-palette-green-a200";
exports.indigo25 = "--cnvs-base-palette-indigo-25";
exports.indigo50 = "--cnvs-base-palette-indigo-50";
exports.indigo100 = "--cnvs-base-palette-indigo-100";
exports.indigo200 = "--cnvs-base-palette-indigo-200";
exports.indigo300 = "--cnvs-base-palette-indigo-300";
exports.indigo400 = "--cnvs-base-palette-indigo-400";
exports.indigo500 = "--cnvs-base-palette-indigo-500";
exports.indigo600 = "--cnvs-base-palette-indigo-600";
exports.indigo700 = "--cnvs-base-palette-indigo-700";
exports.indigo800 = "--cnvs-base-palette-indigo-800";
exports.indigo900 = "--cnvs-base-palette-indigo-900";
exports.indigo950 = "--cnvs-base-palette-indigo-950";
exports.indigo975 = "--cnvs-base-palette-indigo-975";
exports.indigoA25 = "--cnvs-base-palette-indigo-a25";
exports.indigoA50 = "--cnvs-base-palette-indigo-a50";
exports.indigoA100 = "--cnvs-base-palette-indigo-a100";
exports.indigoA200 = "--cnvs-base-palette-indigo-a200";
exports.magenta25 = "--cnvs-base-palette-magenta-25";
exports.magenta50 = "--cnvs-base-palette-magenta-50";
exports.magenta100 = "--cnvs-base-palette-magenta-100";
exports.magenta200 = "--cnvs-base-palette-magenta-200";
exports.magenta300 = "--cnvs-base-palette-magenta-300";
exports.magenta400 = "--cnvs-base-palette-magenta-400";
exports.magenta500 = "--cnvs-base-palette-magenta-500";
exports.magenta600 = "--cnvs-base-palette-magenta-600";
exports.magenta700 = "--cnvs-base-palette-magenta-700";
exports.magenta800 = "--cnvs-base-palette-magenta-800";
exports.magenta900 = "--cnvs-base-palette-magenta-900";
exports.magenta950 = "--cnvs-base-palette-magenta-950";
exports.magenta975 = "--cnvs-base-palette-magenta-975";
exports.magentaA25 = "--cnvs-base-palette-magenta-a25";
exports.magentaA50 = "--cnvs-base-palette-magenta-a50";
exports.magentaA100 = "--cnvs-base-palette-magenta-a100";
exports.magentaA200 = "--cnvs-base-palette-magenta-a200";
exports.neutral0 = "--cnvs-base-palette-neutral-0";
exports.neutral25 = "--cnvs-base-palette-neutral-25";
exports.neutral50 = "--cnvs-base-palette-neutral-50";
exports.neutral100 = "--cnvs-base-palette-neutral-100";
exports.neutral200 = "--cnvs-base-palette-neutral-200";
exports.neutral300 = "--cnvs-base-palette-neutral-300";
exports.neutral400 = "--cnvs-base-palette-neutral-400";
exports.neutral500 = "--cnvs-base-palette-neutral-500";
exports.neutral600 = "--cnvs-base-palette-neutral-600";
exports.neutral700 = "--cnvs-base-palette-neutral-700";
exports.neutral800 = "--cnvs-base-palette-neutral-800";
exports.neutral900 = "--cnvs-base-palette-neutral-900";
exports.neutral950 = "--cnvs-base-palette-neutral-950";
exports.neutral975 = "--cnvs-base-palette-neutral-975";
exports.neutral1000 = "--cnvs-base-palette-neutral-1000";
exports.neutralA0 = "--cnvs-base-palette-neutral-a0";
exports.neutralA25 = "--cnvs-base-palette-neutral-a25";
exports.neutralA50 = "--cnvs-base-palette-neutral-a50";
exports.neutralA100 = "--cnvs-base-palette-neutral-a100";
exports.neutralA200 = "--cnvs-base-palette-neutral-a200";
exports.neutralA300 = "--cnvs-base-palette-neutral-a300";
exports.neutralA400 = "--cnvs-base-palette-neutral-a400";
exports.neutralA500 = "--cnvs-base-palette-neutral-a500";
exports.neutralA600 = "--cnvs-base-palette-neutral-a600";
exports.neutralA700 = "--cnvs-base-palette-neutral-a700";
exports.neutralA800 = "--cnvs-base-palette-neutral-a800";
exports.neutralA900 = "--cnvs-base-palette-neutral-a900";
exports.neutralA950 = "--cnvs-base-palette-neutral-a950";
exports.neutralA975 = "--cnvs-base-palette-neutral-a975";
exports.orange25 = "--cnvs-base-palette-orange-25";
exports.orange50 = "--cnvs-base-palette-orange-50";
exports.orange100 = "--cnvs-base-palette-orange-100";
exports.orange200 = "--cnvs-base-palette-orange-200";
exports.orange300 = "--cnvs-base-palette-orange-300";
exports.orange400 = "--cnvs-base-palette-orange-400";
exports.orange500 = "--cnvs-base-palette-orange-500";
exports.orange600 = "--cnvs-base-palette-orange-600";
exports.orange700 = "--cnvs-base-palette-orange-700";
exports.orange800 = "--cnvs-base-palette-orange-800";
exports.orange900 = "--cnvs-base-palette-orange-900";
exports.orange950 = "--cnvs-base-palette-orange-950";
exports.orange975 = "--cnvs-base-palette-orange-975";
exports.orangeA25 = "--cnvs-base-palette-orange-a25";
exports.orangeA50 = "--cnvs-base-palette-orange-a50";
exports.orangeA100 = "--cnvs-base-palette-orange-a100";
exports.orangeA200 = "--cnvs-base-palette-orange-a200";
exports.purple25 = "--cnvs-base-palette-purple-25";
exports.purple50 = "--cnvs-base-palette-purple-50";
exports.purple100 = "--cnvs-base-palette-purple-100";
exports.purple200 = "--cnvs-base-palette-purple-200";
exports.purple300 = "--cnvs-base-palette-purple-300";
exports.purple400 = "--cnvs-base-palette-purple-400";
exports.purple500 = "--cnvs-base-palette-purple-500";
exports.purple600 = "--cnvs-base-palette-purple-600";
exports.purple700 = "--cnvs-base-palette-purple-700";
exports.purple800 = "--cnvs-base-palette-purple-800";
exports.purple900 = "--cnvs-base-palette-purple-900";
exports.purple950 = "--cnvs-base-palette-purple-950";
exports.purple975 = "--cnvs-base-palette-purple-975";
exports.purpleA25 = "--cnvs-base-palette-purple-a25";
exports.purpleA50 = "--cnvs-base-palette-purple-a50";
exports.purpleA100 = "--cnvs-base-palette-purple-a100";
exports.purpleA200 = "--cnvs-base-palette-purple-a200";
exports.red25 = "--cnvs-base-palette-red-25";
exports.red50 = "--cnvs-base-palette-red-50";
exports.red100 = "--cnvs-base-palette-red-100";
exports.red200 = "--cnvs-base-palette-red-200";
exports.red300 = "--cnvs-base-palette-red-300";
exports.red400 = "--cnvs-base-palette-red-400";
exports.red500 = "--cnvs-base-palette-red-500";
exports.red600 = "--cnvs-base-palette-red-600";
exports.red700 = "--cnvs-base-palette-red-700";
exports.red800 = "--cnvs-base-palette-red-800";
exports.red900 = "--cnvs-base-palette-red-900";
exports.red950 = "--cnvs-base-palette-red-950";
exports.red975 = "--cnvs-base-palette-red-975";
exports.redA25 = "--cnvs-base-palette-red-a25";
exports.redA50 = "--cnvs-base-palette-red-a50";
exports.redA100 = "--cnvs-base-palette-red-a100";
exports.redA200 = "--cnvs-base-palette-red-a200";
exports.slate25 = "--cnvs-base-palette-slate-25";
exports.slate50 = "--cnvs-base-palette-slate-50";
exports.slate100 = "--cnvs-base-palette-slate-100";
exports.slate200 = "--cnvs-base-palette-slate-200";
exports.slate300 = "--cnvs-base-palette-slate-300";
exports.slate400 = "--cnvs-base-palette-slate-400";
exports.slate500 = "--cnvs-base-palette-slate-500";
exports.slate600 = "--cnvs-base-palette-slate-600";
exports.slate700 = "--cnvs-base-palette-slate-700";
exports.slate800 = "--cnvs-base-palette-slate-800";
exports.slate900 = "--cnvs-base-palette-slate-900";
exports.slate950 = "--cnvs-base-palette-slate-950";
exports.slate975 = "--cnvs-base-palette-slate-975";
exports.slateA25 = "--cnvs-base-palette-slate-a25";
exports.slateA50 = "--cnvs-base-palette-slate-a50";
exports.slateA100 = "--cnvs-base-palette-slate-a100";
exports.slateA200 = "--cnvs-base-palette-slate-a200";
exports.slateA300 = "--cnvs-base-palette-slate-a300";
exports.slateA400 = "--cnvs-base-palette-slate-a400";
exports.slateA500 = "--cnvs-base-palette-slate-a500";
exports.slateA600 = "--cnvs-base-palette-slate-a600";
exports.slateA700 = "--cnvs-base-palette-slate-a700";
exports.slateA800 = "--cnvs-base-palette-slate-a800";
exports.slateA900 = "--cnvs-base-palette-slate-a900";
exports.slateA950 = "--cnvs-base-palette-slate-a950";
exports.slateA975 = "--cnvs-base-palette-slate-a975";
exports.teal25 = "--cnvs-base-palette-teal-25";
exports.teal50 = "--cnvs-base-palette-teal-50";
exports.teal100 = "--cnvs-base-palette-teal-100";
exports.teal200 = "--cnvs-base-palette-teal-200";
exports.teal300 = "--cnvs-base-palette-teal-300";
exports.teal400 = "--cnvs-base-palette-teal-400";
exports.teal500 = "--cnvs-base-palette-teal-500";
exports.teal600 = "--cnvs-base-palette-teal-600";
exports.teal700 = "--cnvs-base-palette-teal-700";
exports.teal800 = "--cnvs-base-palette-teal-800";
exports.teal900 = "--cnvs-base-palette-teal-900";
exports.teal950 = "--cnvs-base-palette-teal-950";
exports.teal975 = "--cnvs-base-palette-teal-975";
exports.tealA25 = "--cnvs-base-palette-teal-a25";
exports.tealA50 = "--cnvs-base-palette-teal-a50";
exports.tealA100 = "--cnvs-base-palette-teal-a100";
exports.tealA200 = "--cnvs-base-palette-teal-a200";
exports.whiteA0 = "--cnvs-base-palette-white-a0";
exports.whiteA25 = "--cnvs-base-palette-white-a25";
exports.whiteA50 = "--cnvs-base-palette-white-a50";
exports.whiteA100 = "--cnvs-base-palette-white-a100";
exports.whiteA200 = "--cnvs-base-palette-white-a200";
exports.whiteA300 = "--cnvs-base-palette-white-a300";
exports.whiteA400 = "--cnvs-base-palette-white-a400";
exports.whiteA500 = "--cnvs-base-palette-white-a500";
exports.whiteA600 = "--cnvs-base-palette-white-a600";
exports.whiteA700 = "--cnvs-base-palette-white-a700";
exports.whiteA800 = "--cnvs-base-palette-white-a800";
exports.whiteA900 = "--cnvs-base-palette-white-a900";
exports.whiteA950 = "--cnvs-base-palette-white-a950";
exports.whiteA975 = "--cnvs-base-palette-white-a975";
exports.cinnamon100 = "--cnvs-base-palette-cinnamon-100";
exports.cinnamon200 = "--cnvs-base-palette-cinnamon-200";
exports.cinnamon300 = "--cnvs-base-palette-cinnamon-300";
exports.cinnamon400 = "--cnvs-base-palette-cinnamon-400";
exports.cinnamon500 = "--cnvs-base-palette-cinnamon-500";
exports.cinnamon600 = "--cnvs-base-palette-cinnamon-600";
exports.peach100 = "--cnvs-base-palette-peach-100";
exports.peach200 = "--cnvs-base-palette-peach-200";
exports.peach300 = "--cnvs-base-palette-peach-300";
exports.peach400 = "--cnvs-base-palette-peach-400";
exports.peach500 = "--cnvs-base-palette-peach-500";
exports.peach600 = "--cnvs-base-palette-peach-600";
exports.chiliMango100 = "--cnvs-base-palette-chili-mango-100";
exports.chiliMango200 = "--cnvs-base-palette-chili-mango-200";
exports.chiliMango300 = "--cnvs-base-palette-chili-mango-300";
exports.chiliMango400 = "--cnvs-base-palette-chili-mango-400";
exports.chiliMango500 = "--cnvs-base-palette-chili-mango-500";
exports.chiliMango600 = "--cnvs-base-palette-chili-mango-600";
exports.cantaloupe100 = "--cnvs-base-palette-cantaloupe-100";
exports.cantaloupe200 = "--cnvs-base-palette-cantaloupe-200";
exports.cantaloupe300 = "--cnvs-base-palette-cantaloupe-300";
exports.cantaloupe400 = "--cnvs-base-palette-cantaloupe-400";
exports.cantaloupe500 = "--cnvs-base-palette-cantaloupe-500";
exports.cantaloupe600 = "--cnvs-base-palette-cantaloupe-600";
exports.sourLemon100 = "--cnvs-base-palette-sour-lemon-100";
exports.sourLemon200 = "--cnvs-base-palette-sour-lemon-200";
exports.sourLemon300 = "--cnvs-base-palette-sour-lemon-300";
exports.sourLemon400 = "--cnvs-base-palette-sour-lemon-400";
exports.sourLemon500 = "--cnvs-base-palette-sour-lemon-500";
exports.sourLemon600 = "--cnvs-base-palette-sour-lemon-600";
exports.juicyPear100 = "--cnvs-base-palette-juicy-pear-100";
exports.juicyPear200 = "--cnvs-base-palette-juicy-pear-200";
exports.juicyPear300 = "--cnvs-base-palette-juicy-pear-300";
exports.juicyPear400 = "--cnvs-base-palette-juicy-pear-400";
exports.juicyPear500 = "--cnvs-base-palette-juicy-pear-500";
exports.juicyPear600 = "--cnvs-base-palette-juicy-pear-600";
exports.kiwi100 = "--cnvs-base-palette-kiwi-100";
exports.kiwi200 = "--cnvs-base-palette-kiwi-200";
exports.kiwi300 = "--cnvs-base-palette-kiwi-300";
exports.kiwi400 = "--cnvs-base-palette-kiwi-400";
exports.kiwi500 = "--cnvs-base-palette-kiwi-500";
exports.kiwi600 = "--cnvs-base-palette-kiwi-600";
exports.greenApple100 = "--cnvs-base-palette-green-apple-100";
exports.greenApple200 = "--cnvs-base-palette-green-apple-200";
exports.greenApple300 = "--cnvs-base-palette-green-apple-300";
exports.greenApple400 = "--cnvs-base-palette-green-apple-400";
exports.greenApple500 = "--cnvs-base-palette-green-apple-500";
exports.greenApple600 = "--cnvs-base-palette-green-apple-600";
exports.watermelon100 = "--cnvs-base-palette-watermelon-100";
exports.watermelon200 = "--cnvs-base-palette-watermelon-200";
exports.watermelon300 = "--cnvs-base-palette-watermelon-300";
exports.watermelon400 = "--cnvs-base-palette-watermelon-400";
exports.watermelon500 = "--cnvs-base-palette-watermelon-500";
exports.watermelon600 = "--cnvs-base-palette-watermelon-600";
exports.jewel100 = "--cnvs-base-palette-jewel-100";
exports.jewel200 = "--cnvs-base-palette-jewel-200";
exports.jewel300 = "--cnvs-base-palette-jewel-300";
exports.jewel400 = "--cnvs-base-palette-jewel-400";
exports.jewel500 = "--cnvs-base-palette-jewel-500";
exports.jewel600 = "--cnvs-base-palette-jewel-600";
exports.toothpaste100 = "--cnvs-base-palette-toothpaste-100";
exports.toothpaste200 = "--cnvs-base-palette-toothpaste-200";
exports.toothpaste300 = "--cnvs-base-palette-toothpaste-300";
exports.toothpaste400 = "--cnvs-base-palette-toothpaste-400";
exports.toothpaste500 = "--cnvs-base-palette-toothpaste-500";
exports.toothpaste600 = "--cnvs-base-palette-toothpaste-600";
exports.blueberry100 = "--cnvs-base-palette-blueberry-100";
exports.blueberry200 = "--cnvs-base-palette-blueberry-200";
exports.blueberry300 = "--cnvs-base-palette-blueberry-300";
exports.blueberry400 = "--cnvs-base-palette-blueberry-400";
exports.blueberry500 = "--cnvs-base-palette-blueberry-500";
exports.blueberry600 = "--cnvs-base-palette-blueberry-600";
exports.plum100 = "--cnvs-base-palette-plum-100";
exports.plum200 = "--cnvs-base-palette-plum-200";
exports.plum300 = "--cnvs-base-palette-plum-300";
exports.plum400 = "--cnvs-base-palette-plum-400";
exports.plum500 = "--cnvs-base-palette-plum-500";
exports.plum600 = "--cnvs-base-palette-plum-600";
exports.berrySmoothie100 = "--cnvs-base-palette-berry-smoothie-100";
exports.berrySmoothie200 = "--cnvs-base-palette-berry-smoothie-200";
exports.berrySmoothie300 = "--cnvs-base-palette-berry-smoothie-300";
exports.berrySmoothie400 = "--cnvs-base-palette-berry-smoothie-400";
exports.berrySmoothie500 = "--cnvs-base-palette-berry-smoothie-500";
exports.berrySmoothie600 = "--cnvs-base-palette-berry-smoothie-600";
exports.blackberry100 = "--cnvs-base-palette-blackberry-100";
exports.blackberry200 = "--cnvs-base-palette-blackberry-200";
exports.blackberry300 = "--cnvs-base-palette-blackberry-300";
exports.blackberry400 = "--cnvs-base-palette-blackberry-400";
exports.blackberry500 = "--cnvs-base-palette-blackberry-500";
exports.blackberry600 = "--cnvs-base-palette-blackberry-600";
exports.islandPunch100 = "--cnvs-base-palette-island-punch-100";
exports.islandPunch200 = "--cnvs-base-palette-island-punch-200";
exports.islandPunch300 = "--cnvs-base-palette-island-punch-300";
exports.islandPunch400 = "--cnvs-base-palette-island-punch-400";
exports.islandPunch500 = "--cnvs-base-palette-island-punch-500";
exports.islandPunch600 = "--cnvs-base-palette-island-punch-600";
exports.grapeSoda100 = "--cnvs-base-palette-grape-soda-100";
exports.grapeSoda200 = "--cnvs-base-palette-grape-soda-200";
exports.grapeSoda300 = "--cnvs-base-palette-grape-soda-300";
exports.grapeSoda400 = "--cnvs-base-palette-grape-soda-400";
exports.grapeSoda500 = "--cnvs-base-palette-grape-soda-500";
exports.grapeSoda600 = "--cnvs-base-palette-grape-soda-600";
exports.pomegranate100 = "--cnvs-base-palette-pomegranate-100";
exports.pomegranate200 = "--cnvs-base-palette-pomegranate-200";
exports.pomegranate300 = "--cnvs-base-palette-pomegranate-300";
exports.pomegranate400 = "--cnvs-base-palette-pomegranate-400";
exports.pomegranate500 = "--cnvs-base-palette-pomegranate-500";
exports.pomegranate600 = "--cnvs-base-palette-pomegranate-600";
exports.fruitPunch100 = "--cnvs-base-palette-fruit-punch-100";
exports.fruitPunch200 = "--cnvs-base-palette-fruit-punch-200";
exports.fruitPunch300 = "--cnvs-base-palette-fruit-punch-300";
exports.fruitPunch400 = "--cnvs-base-palette-fruit-punch-400";
exports.fruitPunch500 = "--cnvs-base-palette-fruit-punch-500";
exports.fruitPunch600 = "--cnvs-base-palette-fruit-punch-600";
exports.rootBeer100 = "--cnvs-base-palette-root-beer-100";
exports.rootBeer200 = "--cnvs-base-palette-root-beer-200";
exports.rootBeer300 = "--cnvs-base-palette-root-beer-300";
exports.rootBeer400 = "--cnvs-base-palette-root-beer-400";
exports.rootBeer500 = "--cnvs-base-palette-root-beer-500";
exports.rootBeer600 = "--cnvs-base-palette-root-beer-600";
exports.toastedMarshmallow100 = "--cnvs-base-palette-toasted-marshmallow-100";
exports.toastedMarshmallow200 = "--cnvs-base-palette-toasted-marshmallow-200";
exports.toastedMarshmallow300 = "--cnvs-base-palette-toasted-marshmallow-300";
exports.toastedMarshmallow400 = "--cnvs-base-palette-toasted-marshmallow-400";
exports.toastedMarshmallow500 = "--cnvs-base-palette-toasted-marshmallow-500";
exports.toastedMarshmallow600 = "--cnvs-base-palette-toasted-marshmallow-600";
exports.licorice100 = "--cnvs-base-palette-licorice-100";
exports.licorice200 = "--cnvs-base-palette-licorice-200";
exports.licorice300 = "--cnvs-base-palette-licorice-300";
exports.licorice400 = "--cnvs-base-palette-licorice-400";
exports.licorice500 = "--cnvs-base-palette-licorice-500";
exports.licorice600 = "--cnvs-base-palette-licorice-600";
exports.soap100 = "--cnvs-base-palette-soap-100";
exports.soap200 = "--cnvs-base-palette-soap-200";
exports.soap300 = "--cnvs-base-palette-soap-300";
exports.soap400 = "--cnvs-base-palette-soap-400";
exports.soap500 = "--cnvs-base-palette-soap-500";
exports.soap600 = "--cnvs-base-palette-soap-600";
exports.frenchVanilla100 = "--cnvs-base-palette-french-vanilla-100";
exports.frenchVanilla200 = "--cnvs-base-palette-french-vanilla-200";
exports.frenchVanilla300 = "--cnvs-base-palette-french-vanilla-300";
exports.frenchVanilla400 = "--cnvs-base-palette-french-vanilla-400";
exports.frenchVanilla500 = "--cnvs-base-palette-french-vanilla-500";
exports.frenchVanilla600 = "--cnvs-base-palette-french-vanilla-600";
exports.blackPepper100 = "--cnvs-base-palette-black-pepper-100";
exports.blackPepper200 = "--cnvs-base-palette-black-pepper-200";
exports.blackPepper300 = "--cnvs-base-palette-black-pepper-300";
exports.blackPepper400 = "--cnvs-base-palette-black-pepper-400";
exports.blackPepper500 = "--cnvs-base-palette-black-pepper-500";
exports.blackPepper600 = "--cnvs-base-palette-black-pepper-600";
exports.coconut100 = "--cnvs-base-palette-coconut-100";
exports.coconut200 = "--cnvs-base-palette-coconut-200";
exports.coconut300 = "--cnvs-base-palette-coconut-300";
exports.coconut400 = "--cnvs-base-palette-coconut-400";
exports.coconut500 = "--cnvs-base-palette-coconut-500";
exports.coconut600 = "--cnvs-base-palette-coconut-600";
exports.cappuccino100 = "--cnvs-base-palette-cappuccino-100";
exports.cappuccino200 = "--cnvs-base-palette-cappuccino-200";
exports.cappuccino300 = "--cnvs-base-palette-cappuccino-300";
exports.cappuccino400 = "--cnvs-base-palette-cappuccino-400";
exports.cappuccino500 = "--cnvs-base-palette-cappuccino-500";
exports.cappuccino600 = "--cnvs-base-palette-cappuccino-600";
exports.size0 = "--cnvs-base-size-0";
exports.size25 = "--cnvs-base-size-25";
exports.size50 = "--cnvs-base-size-50";
exports.size75 = "--cnvs-base-size-75";
exports.size100 = "--cnvs-base-size-100";
exports.size125 = "--cnvs-base-size-125";
exports.size150 = "--cnvs-base-size-150";
exports.size175 = "--cnvs-base-size-175";
exports.size200 = "--cnvs-base-size-200";
exports.size225 = "--cnvs-base-size-225";
exports.size250 = "--cnvs-base-size-250";
exports.size300 = "--cnvs-base-size-300";
exports.size350 = "--cnvs-base-size-350";
exports.size400 = "--cnvs-base-size-400";
exports.size450 = "--cnvs-base-size-450";
exports.size500 = "--cnvs-base-size-500";
exports.size600 = "--cnvs-base-size-600";
exports.size700 = "--cnvs-base-size-700";
exports.size800 = "--cnvs-base-size-800";
exports.size900 = "--cnvs-base-size-900";
exports.size1000 = "--cnvs-base-size-1000";
exports.size1100 = "--cnvs-base-size-1100";
exports.size1200 = "--cnvs-base-size-1200";
exports.size1300 = "--cnvs-base-size-1300";
exports.size1400 = "--cnvs-base-size-1400";
exports.fontFamily50 = "--cnvs-base-font-family-50";
exports.fontFamily100 = "--cnvs-base-font-family-100";
exports.fontFamily200 = "--cnvs-base-font-family-200";
exports.fontWeight300 = "--cnvs-base-font-weight-300";
exports.fontWeight400 = "--cnvs-base-font-weight-400";
exports.fontWeight500 = "--cnvs-base-font-weight-500";
exports.fontWeight700 = "--cnvs-base-font-weight-700";
exports.letterSpacing50 = "--cnvs-base-letter-spacing-50";
exports.letterSpacing100 = "--cnvs-base-letter-spacing-100";
exports.letterSpacing150 = "--cnvs-base-letter-spacing-150";
exports.letterSpacing200 = "--cnvs-base-letter-spacing-200";
exports.baseUnit = "--cnvs-base-unit";
exports.extendedDragonFruit100 = "--cnvs-base-extended-palette-dragon-fruit-100";
exports.extendedDragonFruit200 = "--cnvs-base-extended-palette-dragon-fruit-200";
exports.extendedDragonFruit300 = "--cnvs-base-extended-palette-dragon-fruit-300";
exports.extendedDragonFruit400 = "--cnvs-base-extended-palette-dragon-fruit-400";
exports.extendedDragonFruit500 = "--cnvs-base-extended-palette-dragon-fruit-500";
exports.extendedDragonFruit600 = "--cnvs-base-extended-palette-dragon-fruit-600";
exports.fontSize25 = "--cnvs-base-font-size-25";
exports.fontSize50 = "--cnvs-base-font-size-50";
exports.fontSize75 = "--cnvs-base-font-size-75";
exports.fontSize100 = "--cnvs-base-font-size-100";
exports.fontSize125 = "--cnvs-base-font-size-125";
exports.fontSize150 = "--cnvs-base-font-size-150";
exports.fontSize200 = "--cnvs-base-font-size-200";
exports.fontSize250 = "--cnvs-base-font-size-250";
exports.fontSize300 = "--cnvs-base-font-size-300";
exports.fontSize400 = "--cnvs-base-font-size-400";
exports.fontSize500 = "--cnvs-base-font-size-500";
exports.fontSize600 = "--cnvs-base-font-size-600";
exports.fontSize750 = "--cnvs-base-font-size-750";
exports.fontSize900 = "--cnvs-base-font-size-900";
exports.fontSize1050 = "--cnvs-base-font-size-1050";
exports.lineHeight50 = "--cnvs-base-line-height-50";
exports.lineHeight100 = "--cnvs-base-line-height-100";
exports.lineHeight150 = "--cnvs-base-line-height-150";
exports.lineHeight200 = "--cnvs-base-line-height-200";
exports.lineHeight250 = "--cnvs-base-line-height-250";
exports.lineHeight300 = "--cnvs-base-line-height-300";
exports.lineHeight350 = "--cnvs-base-line-height-350";
exports.lineHeight400 = "--cnvs-base-line-height-400";
exports.lineHeight500 = "--cnvs-base-line-height-500";
exports.lineHeight600 = "--cnvs-base-line-height-600";
exports.lineHeight750 = "--cnvs-base-line-height-750";
exports.lineHeight900 = "--cnvs-base-line-height-900";
exports.lineHeight1050 = "--cnvs-base-line-height-1050";
