// @workday/canvas-tokens-web/dist/common-js/brand/index.js
/**
 * Do not edit directly
 * Generated on Wed, 25 Mar 2026 17:29:04 GMT
 */

"use strict";
Object.defineProperty(exports, "__esModule", { value: true });

exports.primary25 = "--cnvs-brand-primary-25";
exports.primary50 = "--cnvs-brand-primary-50";
exports.primary100 = "--cnvs-brand-primary-100";
exports.primary200 = "--cnvs-brand-primary-200";
exports.primary300 = "--cnvs-brand-primary-300";
exports.primary400 = "--cnvs-brand-primary-400";
exports.primary500 = "--cnvs-brand-primary-500";
exports.primary600 = "--cnvs-brand-primary-600";
exports.primary700 = "--cnvs-brand-primary-700";
exports.primary800 = "--cnvs-brand-primary-800";
exports.primary900 = "--cnvs-brand-primary-900";
exports.primary950 = "--cnvs-brand-primary-950";
exports.primary975 = "--cnvs-brand-primary-975";
exports.primary = {
  "base": "--cnvs-brand-primary-base",
  "lightest": "--cnvs-brand-primary-lightest",
  "darkest": "--cnvs-brand-primary-darkest",
  "dark": "--cnvs-brand-primary-dark",
  "accent": "--cnvs-brand-primary-accent",
  "lighter": "--cnvs-brand-primary-lighter",
  "light": "--cnvs-brand-primary-light"
};
exports.primaryA25 = "--cnvs-brand-primary-a25";
exports.primaryA50 = "--cnvs-brand-primary-a50";
exports.primaryA100 = "--cnvs-brand-primary-a100";
exports.primaryA200 = "--cnvs-brand-primary-a200";
exports.error = {
  "base": "--cnvs-brand-error-base",
  "lightest": "--cnvs-brand-error-lightest",
  "dark": "--cnvs-brand-error-dark",
  "darkest": "--cnvs-brand-error-darkest",
  "accent": "--cnvs-brand-error-accent",
  "lighter": "--cnvs-brand-error-lighter",
  "light": "--cnvs-brand-error-light"
};
exports.alert = {
  "base": "--cnvs-brand-alert-base",
  "lighter": "--cnvs-brand-alert-lighter",
  "lightest": "--cnvs-brand-alert-lightest",
  "dark": "--cnvs-brand-alert-dark",
  "darkest": "--cnvs-brand-alert-darkest",
  "accent": "--cnvs-brand-alert-accent",
  "light": "--cnvs-brand-alert-light"
};
exports.success = {
  "base": "--cnvs-brand-success-base",
  "lightest": "--cnvs-brand-success-lightest",
  "lighter": "--cnvs-brand-success-lighter",
  "dark": "--cnvs-brand-success-dark",
  "darkest": "--cnvs-brand-success-darkest",
  "accent": "--cnvs-brand-success-accent",
  "light": "--cnvs-brand-success-light"
};
exports.neutral25 = "--cnvs-brand-neutral-25";
exports.neutral50 = "--cnvs-brand-neutral-50";
exports.neutral100 = "--cnvs-brand-neutral-100";
exports.neutral200 = "--cnvs-brand-neutral-200";
exports.neutral300 = "--cnvs-brand-neutral-300";
exports.neutral400 = "--cnvs-brand-neutral-400";
exports.neutral500 = "--cnvs-brand-neutral-500";
exports.neutral600 = "--cnvs-brand-neutral-600";
exports.neutral700 = "--cnvs-brand-neutral-700";
exports.neutral800 = "--cnvs-brand-neutral-800";
exports.neutral900 = "--cnvs-brand-neutral-900";
exports.neutral950 = "--cnvs-brand-neutral-950";
exports.neutral975 = "--cnvs-brand-neutral-975";
exports.neutral = {
  "lightest": "--cnvs-brand-neutral-lightest",
  "base": "--cnvs-brand-neutral-base",
  "dark": "--cnvs-brand-neutral-dark",
  "darkest": "--cnvs-brand-neutral-darkest",
  "accent": "--cnvs-brand-neutral-accent",
  "lighter": "--cnvs-brand-neutral-lighter",
  "light": "--cnvs-brand-neutral-light"
};
exports.neutralA25 = "--cnvs-brand-neutral-a25";
exports.neutralA50 = "--cnvs-brand-neutral-a50";
exports.neutralA100 = "--cnvs-brand-neutral-a100";
exports.neutralA200 = "--cnvs-brand-neutral-a200";
exports.neutralA300 = "--cnvs-brand-neutral-a300";
exports.neutralA400 = "--cnvs-brand-neutral-a400";
exports.neutralA500 = "--cnvs-brand-neutral-a500";
exports.neutralA600 = "--cnvs-brand-neutral-a600";
exports.neutralA700 = "--cnvs-brand-neutral-a700";
exports.neutralA800 = "--cnvs-brand-neutral-a800";
exports.neutralA900 = "--cnvs-brand-neutral-a900";
exports.neutralA950 = "--cnvs-brand-neutral-a950";
exports.neutralA975 = "--cnvs-brand-neutral-a975";
exports.common = {
  "focusOutline": "--cnvs-brand-common-focus-outline",
  "errorInner": "--cnvs-brand-common-error-inner",
  "alertInner": "--cnvs-brand-common-alert-inner",
  "alertOuter": "--cnvs-brand-common-alert-outer",
  "focus": "--cnvs-brand-common-focus",
  "critical": "--cnvs-brand-common-critical",
  "caution": {
    "inner": "--cnvs-brand-common-caution-inner",
    "outer": "--cnvs-brand-common-caution-outer"
  }
};
exports.critical25 = "--cnvs-brand-critical-25";
exports.critical50 = "--cnvs-brand-critical-50";
exports.critical100 = "--cnvs-brand-critical-100";
exports.critical200 = "--cnvs-brand-critical-200";
exports.critical300 = "--cnvs-brand-critical-300";
exports.critical400 = "--cnvs-brand-critical-400";
exports.critical500 = "--cnvs-brand-critical-500";
exports.critical600 = "--cnvs-brand-critical-600";
exports.critical700 = "--cnvs-brand-critical-700";
exports.critical800 = "--cnvs-brand-critical-800";
exports.critical900 = "--cnvs-brand-critical-900";
exports.critical950 = "--cnvs-brand-critical-950";
exports.critical975 = "--cnvs-brand-critical-975";
exports.criticalA25 = "--cnvs-brand-critical-a25";
exports.criticalA50 = "--cnvs-brand-critical-a50";
exports.criticalA100 = "--cnvs-brand-critical-a100";
exports.criticalA200 = "--cnvs-brand-critical-a200";
exports.caution25 = "--cnvs-brand-caution-25";
exports.caution50 = "--cnvs-brand-caution-50";
exports.caution100 = "--cnvs-brand-caution-100";
exports.caution200 = "--cnvs-brand-caution-200";
exports.caution300 = "--cnvs-brand-caution-300";
exports.caution400 = "--cnvs-brand-caution-400";
exports.caution500 = "--cnvs-brand-caution-500";
exports.caution600 = "--cnvs-brand-caution-600";
exports.caution700 = "--cnvs-brand-caution-700";
exports.caution800 = "--cnvs-brand-caution-800";
exports.caution900 = "--cnvs-brand-caution-900";
exports.caution950 = "--cnvs-brand-caution-950";
exports.caution975 = "--cnvs-brand-caution-975";
exports.cautionA25 = "--cnvs-brand-caution-a25";
exports.cautionA50 = "--cnvs-brand-caution-a50";
exports.cautionA100 = "--cnvs-brand-caution-a100";
exports.cautionA200 = "--cnvs-brand-caution-a200";
exports.positive25 = "--cnvs-brand-positive-25";
exports.positive50 = "--cnvs-brand-positive-50";
exports.positive100 = "--cnvs-brand-positive-100";
exports.positive200 = "--cnvs-brand-positive-200";
exports.positive300 = "--cnvs-brand-positive-300";
exports.positive400 = "--cnvs-brand-positive-400";
exports.positive500 = "--cnvs-brand-positive-500";
exports.positive600 = "--cnvs-brand-positive-600";
exports.positive700 = "--cnvs-brand-positive-700";
exports.positive800 = "--cnvs-brand-positive-800";
exports.positive900 = "--cnvs-brand-positive-900";
exports.positive950 = "--cnvs-brand-positive-950";
exports.positive975 = "--cnvs-brand-positive-975";
exports.positiveA25 = "--cnvs-brand-positive-a25";
exports.positiveA50 = "--cnvs-brand-positive-a50";
exports.positiveA100 = "--cnvs-brand-positive-a100";
exports.positiveA200 = "--cnvs-brand-positive-a200";
exports.action = {
  "base": "--cnvs-brand-action-base",
  "lightest": "--cnvs-brand-action-lightest",
  "light": "--cnvs-brand-action-light",
  "darkest": "--cnvs-brand-action-darkest",
  "dark": "--cnvs-brand-action-dark",
  "accent": "--cnvs-brand-action-accent",
  "darker": "--cnvs-brand-action-darker",
  "lighter": "--cnvs-brand-action-lighter"
};
exports.gradient = {
  "primary": "--cnvs-brand-gradient-primary"
};
