// @workday/canvas-tokens-web/dist/es6/brand/index.js
/**
 * Do not edit directly
 * Generated on Wed, 25 Mar 2026 17:29:03 GMT
 */

export const primary25 = "--cnvs-brand-primary-25";
export const primary50 = "--cnvs-brand-primary-50";
export const primary100 = "--cnvs-brand-primary-100";
export const primary200 = "--cnvs-brand-primary-200";
export const primary300 = "--cnvs-brand-primary-300";
export const primary400 = "--cnvs-brand-primary-400";
export const primary500 = "--cnvs-brand-primary-500";
export const primary600 = "--cnvs-brand-primary-600";
export const primary700 = "--cnvs-brand-primary-700";
export const primary800 = "--cnvs-brand-primary-800";
export const primary900 = "--cnvs-brand-primary-900";
export const primary950 = "--cnvs-brand-primary-950";
export const primary975 = "--cnvs-brand-primary-975";
export const primary = {
  "base": "--cnvs-brand-primary-base",
  "lightest": "--cnvs-brand-primary-lightest",
  "darkest": "--cnvs-brand-primary-darkest",
  "dark": "--cnvs-brand-primary-dark",
  "accent": "--cnvs-brand-primary-accent",
  "lighter": "--cnvs-brand-primary-lighter",
  "light": "--cnvs-brand-primary-light"
};
export const primaryA25 = "--cnvs-brand-primary-a25";
export const primaryA50 = "--cnvs-brand-primary-a50";
export const primaryA100 = "--cnvs-brand-primary-a100";
export const primaryA200 = "--cnvs-brand-primary-a200";
export const error = {
  "base": "--cnvs-brand-error-base",
  "lightest": "--cnvs-brand-error-lightest",
  "dark": "--cnvs-brand-error-dark",
  "darkest": "--cnvs-brand-error-darkest",
  "accent": "--cnvs-brand-error-accent",
  "lighter": "--cnvs-brand-error-lighter",
  "light": "--cnvs-brand-error-light"
};
export const alert = {
  "base": "--cnvs-brand-alert-base",
  "lighter": "--cnvs-brand-alert-lighter",
  "lightest": "--cnvs-brand-alert-lightest",
  "dark": "--cnvs-brand-alert-dark",
  "darkest": "--cnvs-brand-alert-darkest",
  "accent": "--cnvs-brand-alert-accent",
  "light": "--cnvs-brand-alert-light"
};
export const success = {
  "base": "--cnvs-brand-success-base",
  "lightest": "--cnvs-brand-success-lightest",
  "lighter": "--cnvs-brand-success-lighter",
  "dark": "--cnvs-brand-success-dark",
  "darkest": "--cnvs-brand-success-darkest",
  "accent": "--cnvs-brand-success-accent",
  "light": "--cnvs-brand-success-light"
};
export const neutral25 = "--cnvs-brand-neutral-25";
export const neutral50 = "--cnvs-brand-neutral-50";
export const neutral100 = "--cnvs-brand-neutral-100";
export const neutral200 = "--cnvs-brand-neutral-200";
export const neutral300 = "--cnvs-brand-neutral-300";
export const neutral400 = "--cnvs-brand-neutral-400";
export const neutral500 = "--cnvs-brand-neutral-500";
export const neutral600 = "--cnvs-brand-neutral-600";
export const neutral700 = "--cnvs-brand-neutral-700";
export const neutral800 = "--cnvs-brand-neutral-800";
export const neutral900 = "--cnvs-brand-neutral-900";
export const neutral950 = "--cnvs-brand-neutral-950";
export const neutral975 = "--cnvs-brand-neutral-975";
export const neutral = {
  "lightest": "--cnvs-brand-neutral-lightest",
  "base": "--cnvs-brand-neutral-base",
  "dark": "--cnvs-brand-neutral-dark",
  "darkest": "--cnvs-brand-neutral-darkest",
  "accent": "--cnvs-brand-neutral-accent",
  "lighter": "--cnvs-brand-neutral-lighter",
  "light": "--cnvs-brand-neutral-light"
};
export const neutralA25 = "--cnvs-brand-neutral-a25";
export const neutralA50 = "--cnvs-brand-neutral-a50";
export const neutralA100 = "--cnvs-brand-neutral-a100";
export const neutralA200 = "--cnvs-brand-neutral-a200";
export const neutralA300 = "--cnvs-brand-neutral-a300";
export const neutralA400 = "--cnvs-brand-neutral-a400";
export const neutralA500 = "--cnvs-brand-neutral-a500";
export const neutralA600 = "--cnvs-brand-neutral-a600";
export const neutralA700 = "--cnvs-brand-neutral-a700";
export const neutralA800 = "--cnvs-brand-neutral-a800";
export const neutralA900 = "--cnvs-brand-neutral-a900";
export const neutralA950 = "--cnvs-brand-neutral-a950";
export const neutralA975 = "--cnvs-brand-neutral-a975";
export const common = {
  "focusOutline": "--cnvs-brand-common-focus-outline",
  "errorInner": "--cnvs-brand-common-error-inner",
  "alertInner": "--cnvs-brand-common-alert-inner",
  "alertOuter": "--cnvs-brand-common-alert-outer",
  "focus": "--cnvs-brand-common-focus",
  "critical": "--cnvs-brand-common-critical",
  "caution": {
    "inner": "--cnvs-brand-common-caution-inner",
    "outer": "--cnvs-brand-common-caution-outer"
  }
};
export const critical25 = "--cnvs-brand-critical-25";
export const critical50 = "--cnvs-brand-critical-50";
export const critical100 = "--cnvs-brand-critical-100";
export const critical200 = "--cnvs-brand-critical-200";
export const critical300 = "--cnvs-brand-critical-300";
export const critical400 = "--cnvs-brand-critical-400";
export const critical500 = "--cnvs-brand-critical-500";
export const critical600 = "--cnvs-brand-critical-600";
export const critical700 = "--cnvs-brand-critical-700";
export const critical800 = "--cnvs-brand-critical-800";
export const critical900 = "--cnvs-brand-critical-900";
export const critical950 = "--cnvs-brand-critical-950";
export const critical975 = "--cnvs-brand-critical-975";
export const criticalA25 = "--cnvs-brand-critical-a25";
export const criticalA50 = "--cnvs-brand-critical-a50";
export const criticalA100 = "--cnvs-brand-critical-a100";
export const criticalA200 = "--cnvs-brand-critical-a200";
export const caution25 = "--cnvs-brand-caution-25";
export const caution50 = "--cnvs-brand-caution-50";
export const caution100 = "--cnvs-brand-caution-100";
export const caution200 = "--cnvs-brand-caution-200";
export const caution300 = "--cnvs-brand-caution-300";
export const caution400 = "--cnvs-brand-caution-400";
export const caution500 = "--cnvs-brand-caution-500";
export const caution600 = "--cnvs-brand-caution-600";
export const caution700 = "--cnvs-brand-caution-700";
export const caution800 = "--cnvs-brand-caution-800";
export const caution900 = "--cnvs-brand-caution-900";
export const caution950 = "--cnvs-brand-caution-950";
export const caution975 = "--cnvs-brand-caution-975";
export const cautionA25 = "--cnvs-brand-caution-a25";
export const cautionA50 = "--cnvs-brand-caution-a50";
export const cautionA100 = "--cnvs-brand-caution-a100";
export const cautionA200 = "--cnvs-brand-caution-a200";
export const positive25 = "--cnvs-brand-positive-25";
export const positive50 = "--cnvs-brand-positive-50";
export const positive100 = "--cnvs-brand-positive-100";
export const positive200 = "--cnvs-brand-positive-200";
export const positive300 = "--cnvs-brand-positive-300";
export const positive400 = "--cnvs-brand-positive-400";
export const positive500 = "--cnvs-brand-positive-500";
export const positive600 = "--cnvs-brand-positive-600";
export const positive700 = "--cnvs-brand-positive-700";
export const positive800 = "--cnvs-brand-positive-800";
export const positive900 = "--cnvs-brand-positive-900";
export const positive950 = "--cnvs-brand-positive-950";
export const positive975 = "--cnvs-brand-positive-975";
export const positiveA25 = "--cnvs-brand-positive-a25";
export const positiveA50 = "--cnvs-brand-positive-a50";
export const positiveA100 = "--cnvs-brand-positive-a100";
export const positiveA200 = "--cnvs-brand-positive-a200";
export const action = {
  "base": "--cnvs-brand-action-base",
  "lightest": "--cnvs-brand-action-lightest",
  "light": "--cnvs-brand-action-light",
  "darkest": "--cnvs-brand-action-darkest",
  "dark": "--cnvs-brand-action-dark",
  "accent": "--cnvs-brand-action-accent",
  "darker": "--cnvs-brand-action-darker",
  "lighter": "--cnvs-brand-action-lighter"
};
export const gradient = {
  "primary": "--cnvs-brand-gradient-primary"
};
