// @workday/canvas-tokens-web/dist/es6/base/index.js
/**
 * Do not edit directly
 * Generated on Wed, 25 Mar 2026 17:29:03 GMT
 */

export const fontSize = "--cnvs-base-font-size";
export const baseline = "--cnvs-base-baseline";
export const duration50 = "--cnvs-base-duration-50";
export const duration100 = "--cnvs-base-duration-100";
export const duration150 = "--cnvs-base-duration-150";
export const duration200 = "--cnvs-base-duration-200";
export const duration250 = "--cnvs-base-duration-250";
export const duration300 = "--cnvs-base-duration-300";
export const duration350 = "--cnvs-base-duration-350";
export const duration400 = "--cnvs-base-duration-400";
export const duration450 = "--cnvs-base-duration-450";
export const duration500 = "--cnvs-base-duration-500";
export const duration550 = "--cnvs-base-duration-550";
export const duration600 = "--cnvs-base-duration-600";
export const duration650 = "--cnvs-base-duration-650";
export const duration700 = "--cnvs-base-duration-700";
export const duration750 = "--cnvs-base-duration-750";
export const duration800 = "--cnvs-base-duration-800";
export const duration850 = "--cnvs-base-duration-850";
export const duration900 = "--cnvs-base-duration-900";
export const duration950 = "--cnvs-base-duration-950";
export const duration1000 = "--cnvs-base-duration-1000";
export const easingA100 = "--cnvs-base-easing-a-100";
export const easingA200 = "--cnvs-base-easing-a-200";
export const easingA300 = "--cnvs-base-easing-a-300";
export const easingB100 = "--cnvs-base-easing-b-100";
export const easingB200 = "--cnvs-base-easing-b-200";
export const easingB300 = "--cnvs-base-easing-b-300";
export const opacity0 = "--cnvs-base-opacity-0";
export const opacity80 = "--cnvs-base-opacity-80";
export const opacity100 = "--cnvs-base-opacity-100";
export const opacity120 = "--cnvs-base-opacity-120";
export const opacity180 = "--cnvs-base-opacity-180";
export const opacity200 = "--cnvs-base-opacity-200";
export const opacity250 = "--cnvs-base-opacity-250";
export const opacity300 = "--cnvs-base-opacity-300";
export const opacity360 = "--cnvs-base-opacity-360";
export const opacity400 = "--cnvs-base-opacity-400";
export const opacity500 = "--cnvs-base-opacity-500";
export const opacity640 = "--cnvs-base-opacity-640";
export const opacity840 = "--cnvs-base-opacity-840";
export const amber25 = "--cnvs-base-palette-amber-25";
export const amber50 = "--cnvs-base-palette-amber-50";
export const amber100 = "--cnvs-base-palette-amber-100";
export const amber200 = "--cnvs-base-palette-amber-200";
export const amber300 = "--cnvs-base-palette-amber-300";
export const amber400 = "--cnvs-base-palette-amber-400";
export const amber500 = "--cnvs-base-palette-amber-500";
export const amber600 = "--cnvs-base-palette-amber-600";
export const amber700 = "--cnvs-base-palette-amber-700";
export const amber800 = "--cnvs-base-palette-amber-800";
export const amber900 = "--cnvs-base-palette-amber-900";
export const amber950 = "--cnvs-base-palette-amber-950";
export const amber975 = "--cnvs-base-palette-amber-975";
export const amberA25 = "--cnvs-base-palette-amber-a25";
export const amberA50 = "--cnvs-base-palette-amber-a50";
export const amberA100 = "--cnvs-base-palette-amber-a100";
export const amberA200 = "--cnvs-base-palette-amber-a200";
export const azure25 = "--cnvs-base-palette-azure-25";
export const azure50 = "--cnvs-base-palette-azure-50";
export const azure100 = "--cnvs-base-palette-azure-100";
export const azure200 = "--cnvs-base-palette-azure-200";
export const azure300 = "--cnvs-base-palette-azure-300";
export const azure400 = "--cnvs-base-palette-azure-400";
export const azure500 = "--cnvs-base-palette-azure-500";
export const azure600 = "--cnvs-base-palette-azure-600";
export const azure700 = "--cnvs-base-palette-azure-700";
export const azure800 = "--cnvs-base-palette-azure-800";
export const azure900 = "--cnvs-base-palette-azure-900";
export const azure950 = "--cnvs-base-palette-azure-950";
export const azure975 = "--cnvs-base-palette-azure-975";
export const azureA25 = "--cnvs-base-palette-azure-a25";
export const azureA50 = "--cnvs-base-palette-azure-a50";
export const azureA100 = "--cnvs-base-palette-azure-a100";
export const azureA200 = "--cnvs-base-palette-azure-a200";
export const blue25 = "--cnvs-base-palette-blue-25";
export const blue50 = "--cnvs-base-palette-blue-50";
export const blue100 = "--cnvs-base-palette-blue-100";
export const blue200 = "--cnvs-base-palette-blue-200";
export const blue300 = "--cnvs-base-palette-blue-300";
export const blue400 = "--cnvs-base-palette-blue-400";
export const blue500 = "--cnvs-base-palette-blue-500";
export const blue600 = "--cnvs-base-palette-blue-600";
export const blue700 = "--cnvs-base-palette-blue-700";
export const blue800 = "--cnvs-base-palette-blue-800";
export const blue900 = "--cnvs-base-palette-blue-900";
export const blue950 = "--cnvs-base-palette-blue-950";
export const blue975 = "--cnvs-base-palette-blue-975";
export const blueA25 = "--cnvs-base-palette-blue-a25";
export const blueA50 = "--cnvs-base-palette-blue-a50";
export const blueA100 = "--cnvs-base-palette-blue-a100";
export const blueA200 = "--cnvs-base-palette-blue-a200";
export const coral25 = "--cnvs-base-palette-coral-25";
export const coral50 = "--cnvs-base-palette-coral-50";
export const coral100 = "--cnvs-base-palette-coral-100";
export const coral200 = "--cnvs-base-palette-coral-200";
export const coral300 = "--cnvs-base-palette-coral-300";
export const coral400 = "--cnvs-base-palette-coral-400";
export const coral500 = "--cnvs-base-palette-coral-500";
export const coral600 = "--cnvs-base-palette-coral-600";
export const coral700 = "--cnvs-base-palette-coral-700";
export const coral800 = "--cnvs-base-palette-coral-800";
export const coral900 = "--cnvs-base-palette-coral-900";
export const coral950 = "--cnvs-base-palette-coral-950";
export const coral975 = "--cnvs-base-palette-coral-975";
export const coralA25 = "--cnvs-base-palette-coral-a25";
export const coralA50 = "--cnvs-base-palette-coral-a50";
export const coralA100 = "--cnvs-base-palette-coral-a100";
export const coralA200 = "--cnvs-base-palette-coral-a200";
export const green25 = "--cnvs-base-palette-green-25";
export const green50 = "--cnvs-base-palette-green-50";
export const green100 = "--cnvs-base-palette-green-100";
export const green200 = "--cnvs-base-palette-green-200";
export const green300 = "--cnvs-base-palette-green-300";
export const green400 = "--cnvs-base-palette-green-400";
export const green500 = "--cnvs-base-palette-green-500";
export const green600 = "--cnvs-base-palette-green-600";
export const green700 = "--cnvs-base-palette-green-700";
export const green800 = "--cnvs-base-palette-green-800";
export const green900 = "--cnvs-base-palette-green-900";
export const green950 = "--cnvs-base-palette-green-950";
export const green975 = "--cnvs-base-palette-green-975";
export const greenA25 = "--cnvs-base-palette-green-a25";
export const greenA50 = "--cnvs-base-palette-green-a50";
export const greenA100 = "--cnvs-base-palette-green-a100";
export const greenA200 = "--cnvs-base-palette-green-a200";
export const indigo25 = "--cnvs-base-palette-indigo-25";
export const indigo50 = "--cnvs-base-palette-indigo-50";
export const indigo100 = "--cnvs-base-palette-indigo-100";
export const indigo200 = "--cnvs-base-palette-indigo-200";
export const indigo300 = "--cnvs-base-palette-indigo-300";
export const indigo400 = "--cnvs-base-palette-indigo-400";
export const indigo500 = "--cnvs-base-palette-indigo-500";
export const indigo600 = "--cnvs-base-palette-indigo-600";
export const indigo700 = "--cnvs-base-palette-indigo-700";
export const indigo800 = "--cnvs-base-palette-indigo-800";
export const indigo900 = "--cnvs-base-palette-indigo-900";
export const indigo950 = "--cnvs-base-palette-indigo-950";
export const indigo975 = "--cnvs-base-palette-indigo-975";
export const indigoA25 = "--cnvs-base-palette-indigo-a25";
export const indigoA50 = "--cnvs-base-palette-indigo-a50";
export const indigoA100 = "--cnvs-base-palette-indigo-a100";
export const indigoA200 = "--cnvs-base-palette-indigo-a200";
export const magenta25 = "--cnvs-base-palette-magenta-25";
export const magenta50 = "--cnvs-base-palette-magenta-50";
export const magenta100 = "--cnvs-base-palette-magenta-100";
export const magenta200 = "--cnvs-base-palette-magenta-200";
export const magenta300 = "--cnvs-base-palette-magenta-300";
export const magenta400 = "--cnvs-base-palette-magenta-400";
export const magenta500 = "--cnvs-base-palette-magenta-500";
export const magenta600 = "--cnvs-base-palette-magenta-600";
export const magenta700 = "--cnvs-base-palette-magenta-700";
export const magenta800 = "--cnvs-base-palette-magenta-800";
export const magenta900 = "--cnvs-base-palette-magenta-900";
export const magenta950 = "--cnvs-base-palette-magenta-950";
export const magenta975 = "--cnvs-base-palette-magenta-975";
export const magentaA25 = "--cnvs-base-palette-magenta-a25";
export const magentaA50 = "--cnvs-base-palette-magenta-a50";
export const magentaA100 = "--cnvs-base-palette-magenta-a100";
export const magentaA200 = "--cnvs-base-palette-magenta-a200";
export const neutral0 = "--cnvs-base-palette-neutral-0";
export const neutral25 = "--cnvs-base-palette-neutral-25";
export const neutral50 = "--cnvs-base-palette-neutral-50";
export const neutral100 = "--cnvs-base-palette-neutral-100";
export const neutral200 = "--cnvs-base-palette-neutral-200";
export const neutral300 = "--cnvs-base-palette-neutral-300";
export const neutral400 = "--cnvs-base-palette-neutral-400";
export const neutral500 = "--cnvs-base-palette-neutral-500";
export const neutral600 = "--cnvs-base-palette-neutral-600";
export const neutral700 = "--cnvs-base-palette-neutral-700";
export const neutral800 = "--cnvs-base-palette-neutral-800";
export const neutral900 = "--cnvs-base-palette-neutral-900";
export const neutral950 = "--cnvs-base-palette-neutral-950";
export const neutral975 = "--cnvs-base-palette-neutral-975";
export const neutral1000 = "--cnvs-base-palette-neutral-1000";
export const neutralA0 = "--cnvs-base-palette-neutral-a0";
export const neutralA25 = "--cnvs-base-palette-neutral-a25";
export const neutralA50 = "--cnvs-base-palette-neutral-a50";
export const neutralA100 = "--cnvs-base-palette-neutral-a100";
export const neutralA200 = "--cnvs-base-palette-neutral-a200";
export const neutralA300 = "--cnvs-base-palette-neutral-a300";
export const neutralA400 = "--cnvs-base-palette-neutral-a400";
export const neutralA500 = "--cnvs-base-palette-neutral-a500";
export const neutralA600 = "--cnvs-base-palette-neutral-a600";
export const neutralA700 = "--cnvs-base-palette-neutral-a700";
export const neutralA800 = "--cnvs-base-palette-neutral-a800";
export const neutralA900 = "--cnvs-base-palette-neutral-a900";
export const neutralA950 = "--cnvs-base-palette-neutral-a950";
export const neutralA975 = "--cnvs-base-palette-neutral-a975";
export const orange25 = "--cnvs-base-palette-orange-25";
export const orange50 = "--cnvs-base-palette-orange-50";
export const orange100 = "--cnvs-base-palette-orange-100";
export const orange200 = "--cnvs-base-palette-orange-200";
export const orange300 = "--cnvs-base-palette-orange-300";
export const orange400 = "--cnvs-base-palette-orange-400";
export const orange500 = "--cnvs-base-palette-orange-500";
export const orange600 = "--cnvs-base-palette-orange-600";
export const orange700 = "--cnvs-base-palette-orange-700";
export const orange800 = "--cnvs-base-palette-orange-800";
export const orange900 = "--cnvs-base-palette-orange-900";
export const orange950 = "--cnvs-base-palette-orange-950";
export const orange975 = "--cnvs-base-palette-orange-975";
export const orangeA25 = "--cnvs-base-palette-orange-a25";
export const orangeA50 = "--cnvs-base-palette-orange-a50";
export const orangeA100 = "--cnvs-base-palette-orange-a100";
export const orangeA200 = "--cnvs-base-palette-orange-a200";
export const purple25 = "--cnvs-base-palette-purple-25";
export const purple50 = "--cnvs-base-palette-purple-50";
export const purple100 = "--cnvs-base-palette-purple-100";
export const purple200 = "--cnvs-base-palette-purple-200";
export const purple300 = "--cnvs-base-palette-purple-300";
export const purple400 = "--cnvs-base-palette-purple-400";
export const purple500 = "--cnvs-base-palette-purple-500";
export const purple600 = "--cnvs-base-palette-purple-600";
export const purple700 = "--cnvs-base-palette-purple-700";
export const purple800 = "--cnvs-base-palette-purple-800";
export const purple900 = "--cnvs-base-palette-purple-900";
export const purple950 = "--cnvs-base-palette-purple-950";
export const purple975 = "--cnvs-base-palette-purple-975";
export const purpleA25 = "--cnvs-base-palette-purple-a25";
export const purpleA50 = "--cnvs-base-palette-purple-a50";
export const purpleA100 = "--cnvs-base-palette-purple-a100";
export const purpleA200 = "--cnvs-base-palette-purple-a200";
export const red25 = "--cnvs-base-palette-red-25";
export const red50 = "--cnvs-base-palette-red-50";
export const red100 = "--cnvs-base-palette-red-100";
export const red200 = "--cnvs-base-palette-red-200";
export const red300 = "--cnvs-base-palette-red-300";
export const red400 = "--cnvs-base-palette-red-400";
export const red500 = "--cnvs-base-palette-red-500";
export const red600 = "--cnvs-base-palette-red-600";
export const red700 = "--cnvs-base-palette-red-700";
export const red800 = "--cnvs-base-palette-red-800";
export const red900 = "--cnvs-base-palette-red-900";
export const red950 = "--cnvs-base-palette-red-950";
export const red975 = "--cnvs-base-palette-red-975";
export const redA25 = "--cnvs-base-palette-red-a25";
export const redA50 = "--cnvs-base-palette-red-a50";
export const redA100 = "--cnvs-base-palette-red-a100";
export const redA200 = "--cnvs-base-palette-red-a200";
export const slate25 = "--cnvs-base-palette-slate-25";
export const slate50 = "--cnvs-base-palette-slate-50";
export const slate100 = "--cnvs-base-palette-slate-100";
export const slate200 = "--cnvs-base-palette-slate-200";
export const slate300 = "--cnvs-base-palette-slate-300";
export const slate400 = "--cnvs-base-palette-slate-400";
export const slate500 = "--cnvs-base-palette-slate-500";
export const slate600 = "--cnvs-base-palette-slate-600";
export const slate700 = "--cnvs-base-palette-slate-700";
export const slate800 = "--cnvs-base-palette-slate-800";
export const slate900 = "--cnvs-base-palette-slate-900";
export const slate950 = "--cnvs-base-palette-slate-950";
export const slate975 = "--cnvs-base-palette-slate-975";
export const slateA25 = "--cnvs-base-palette-slate-a25";
export const slateA50 = "--cnvs-base-palette-slate-a50";
export const slateA100 = "--cnvs-base-palette-slate-a100";
export const slateA200 = "--cnvs-base-palette-slate-a200";
export const slateA300 = "--cnvs-base-palette-slate-a300";
export const slateA400 = "--cnvs-base-palette-slate-a400";
export const slateA500 = "--cnvs-base-palette-slate-a500";
export const slateA600 = "--cnvs-base-palette-slate-a600";
export const slateA700 = "--cnvs-base-palette-slate-a700";
export const slateA800 = "--cnvs-base-palette-slate-a800";
export const slateA900 = "--cnvs-base-palette-slate-a900";
export const slateA950 = "--cnvs-base-palette-slate-a950";
export const slateA975 = "--cnvs-base-palette-slate-a975";
export const teal25 = "--cnvs-base-palette-teal-25";
export const teal50 = "--cnvs-base-palette-teal-50";
export const teal100 = "--cnvs-base-palette-teal-100";
export const teal200 = "--cnvs-base-palette-teal-200";
export const teal300 = "--cnvs-base-palette-teal-300";
export const teal400 = "--cnvs-base-palette-teal-400";
export const teal500 = "--cnvs-base-palette-teal-500";
export const teal600 = "--cnvs-base-palette-teal-600";
export const teal700 = "--cnvs-base-palette-teal-700";
export const teal800 = "--cnvs-base-palette-teal-800";
export const teal900 = "--cnvs-base-palette-teal-900";
export const teal950 = "--cnvs-base-palette-teal-950";
export const teal975 = "--cnvs-base-palette-teal-975";
export const tealA25 = "--cnvs-base-palette-teal-a25";
export const tealA50 = "--cnvs-base-palette-teal-a50";
export const tealA100 = "--cnvs-base-palette-teal-a100";
export const tealA200 = "--cnvs-base-palette-teal-a200";
export const whiteA0 = "--cnvs-base-palette-white-a0";
export const whiteA25 = "--cnvs-base-palette-white-a25";
export const whiteA50 = "--cnvs-base-palette-white-a50";
export const whiteA100 = "--cnvs-base-palette-white-a100";
export const whiteA200 = "--cnvs-base-palette-white-a200";
export const whiteA300 = "--cnvs-base-palette-white-a300";
export const whiteA400 = "--cnvs-base-palette-white-a400";
export const whiteA500 = "--cnvs-base-palette-white-a500";
export const whiteA600 = "--cnvs-base-palette-white-a600";
export const whiteA700 = "--cnvs-base-palette-white-a700";
export const whiteA800 = "--cnvs-base-palette-white-a800";
export const whiteA900 = "--cnvs-base-palette-white-a900";
export const whiteA950 = "--cnvs-base-palette-white-a950";
export const whiteA975 = "--cnvs-base-palette-white-a975";
export const cinnamon100 = "--cnvs-base-palette-cinnamon-100";
export const cinnamon200 = "--cnvs-base-palette-cinnamon-200";
export const cinnamon300 = "--cnvs-base-palette-cinnamon-300";
export const cinnamon400 = "--cnvs-base-palette-cinnamon-400";
export const cinnamon500 = "--cnvs-base-palette-cinnamon-500";
export const cinnamon600 = "--cnvs-base-palette-cinnamon-600";
export const peach100 = "--cnvs-base-palette-peach-100";
export const peach200 = "--cnvs-base-palette-peach-200";
export const peach300 = "--cnvs-base-palette-peach-300";
export const peach400 = "--cnvs-base-palette-peach-400";
export const peach500 = "--cnvs-base-palette-peach-500";
export const peach600 = "--cnvs-base-palette-peach-600";
export const chiliMango100 = "--cnvs-base-palette-chili-mango-100";
export const chiliMango200 = "--cnvs-base-palette-chili-mango-200";
export const chiliMango300 = "--cnvs-base-palette-chili-mango-300";
export const chiliMango400 = "--cnvs-base-palette-chili-mango-400";
export const chiliMango500 = "--cnvs-base-palette-chili-mango-500";
export const chiliMango600 = "--cnvs-base-palette-chili-mango-600";
export const cantaloupe100 = "--cnvs-base-palette-cantaloupe-100";
export const cantaloupe200 = "--cnvs-base-palette-cantaloupe-200";
export const cantaloupe300 = "--cnvs-base-palette-cantaloupe-300";
export const cantaloupe400 = "--cnvs-base-palette-cantaloupe-400";
export const cantaloupe500 = "--cnvs-base-palette-cantaloupe-500";
export const cantaloupe600 = "--cnvs-base-palette-cantaloupe-600";
export const sourLemon100 = "--cnvs-base-palette-sour-lemon-100";
export const sourLemon200 = "--cnvs-base-palette-sour-lemon-200";
export const sourLemon300 = "--cnvs-base-palette-sour-lemon-300";
export const sourLemon400 = "--cnvs-base-palette-sour-lemon-400";
export const sourLemon500 = "--cnvs-base-palette-sour-lemon-500";
export const sourLemon600 = "--cnvs-base-palette-sour-lemon-600";
export const juicyPear100 = "--cnvs-base-palette-juicy-pear-100";
export const juicyPear200 = "--cnvs-base-palette-juicy-pear-200";
export const juicyPear300 = "--cnvs-base-palette-juicy-pear-300";
export const juicyPear400 = "--cnvs-base-palette-juicy-pear-400";
export const juicyPear500 = "--cnvs-base-palette-juicy-pear-500";
export const juicyPear600 = "--cnvs-base-palette-juicy-pear-600";
export const kiwi100 = "--cnvs-base-palette-kiwi-100";
export const kiwi200 = "--cnvs-base-palette-kiwi-200";
export const kiwi300 = "--cnvs-base-palette-kiwi-300";
export const kiwi400 = "--cnvs-base-palette-kiwi-400";
export const kiwi500 = "--cnvs-base-palette-kiwi-500";
export const kiwi600 = "--cnvs-base-palette-kiwi-600";
export const greenApple100 = "--cnvs-base-palette-green-apple-100";
export const greenApple200 = "--cnvs-base-palette-green-apple-200";
export const greenApple300 = "--cnvs-base-palette-green-apple-300";
export const greenApple400 = "--cnvs-base-palette-green-apple-400";
export const greenApple500 = "--cnvs-base-palette-green-apple-500";
export const greenApple600 = "--cnvs-base-palette-green-apple-600";
export const watermelon100 = "--cnvs-base-palette-watermelon-100";
export const watermelon200 = "--cnvs-base-palette-watermelon-200";
export const watermelon300 = "--cnvs-base-palette-watermelon-300";
export const watermelon400 = "--cnvs-base-palette-watermelon-400";
export const watermelon500 = "--cnvs-base-palette-watermelon-500";
export const watermelon600 = "--cnvs-base-palette-watermelon-600";
export const jewel100 = "--cnvs-base-palette-jewel-100";
export const jewel200 = "--cnvs-base-palette-jewel-200";
export const jewel300 = "--cnvs-base-palette-jewel-300";
export const jewel400 = "--cnvs-base-palette-jewel-400";
export const jewel500 = "--cnvs-base-palette-jewel-500";
export const jewel600 = "--cnvs-base-palette-jewel-600";
export const toothpaste100 = "--cnvs-base-palette-toothpaste-100";
export const toothpaste200 = "--cnvs-base-palette-toothpaste-200";
export const toothpaste300 = "--cnvs-base-palette-toothpaste-300";
export const toothpaste400 = "--cnvs-base-palette-toothpaste-400";
export const toothpaste500 = "--cnvs-base-palette-toothpaste-500";
export const toothpaste600 = "--cnvs-base-palette-toothpaste-600";
export const blueberry100 = "--cnvs-base-palette-blueberry-100";
export const blueberry200 = "--cnvs-base-palette-blueberry-200";
export const blueberry300 = "--cnvs-base-palette-blueberry-300";
export const blueberry400 = "--cnvs-base-palette-blueberry-400";
export const blueberry500 = "--cnvs-base-palette-blueberry-500";
export const blueberry600 = "--cnvs-base-palette-blueberry-600";
export const plum100 = "--cnvs-base-palette-plum-100";
export const plum200 = "--cnvs-base-palette-plum-200";
export const plum300 = "--cnvs-base-palette-plum-300";
export const plum400 = "--cnvs-base-palette-plum-400";
export const plum500 = "--cnvs-base-palette-plum-500";
export const plum600 = "--cnvs-base-palette-plum-600";
export const berrySmoothie100 = "--cnvs-base-palette-berry-smoothie-100";
export const berrySmoothie200 = "--cnvs-base-palette-berry-smoothie-200";
export const berrySmoothie300 = "--cnvs-base-palette-berry-smoothie-300";
export const berrySmoothie400 = "--cnvs-base-palette-berry-smoothie-400";
export const berrySmoothie500 = "--cnvs-base-palette-berry-smoothie-500";
export const berrySmoothie600 = "--cnvs-base-palette-berry-smoothie-600";
export const blackberry100 = "--cnvs-base-palette-blackberry-100";
export const blackberry200 = "--cnvs-base-palette-blackberry-200";
export const blackberry300 = "--cnvs-base-palette-blackberry-300";
export const blackberry400 = "--cnvs-base-palette-blackberry-400";
export const blackberry500 = "--cnvs-base-palette-blackberry-500";
export const blackberry600 = "--cnvs-base-palette-blackberry-600";
export const islandPunch100 = "--cnvs-base-palette-island-punch-100";
export const islandPunch200 = "--cnvs-base-palette-island-punch-200";
export const islandPunch300 = "--cnvs-base-palette-island-punch-300";
export const islandPunch400 = "--cnvs-base-palette-island-punch-400";
export const islandPunch500 = "--cnvs-base-palette-island-punch-500";
export const islandPunch600 = "--cnvs-base-palette-island-punch-600";
export const grapeSoda100 = "--cnvs-base-palette-grape-soda-100";
export const grapeSoda200 = "--cnvs-base-palette-grape-soda-200";
export const grapeSoda300 = "--cnvs-base-palette-grape-soda-300";
export const grapeSoda400 = "--cnvs-base-palette-grape-soda-400";
export const grapeSoda500 = "--cnvs-base-palette-grape-soda-500";
export const grapeSoda600 = "--cnvs-base-palette-grape-soda-600";
export const pomegranate100 = "--cnvs-base-palette-pomegranate-100";
export const pomegranate200 = "--cnvs-base-palette-pomegranate-200";
export const pomegranate300 = "--cnvs-base-palette-pomegranate-300";
export const pomegranate400 = "--cnvs-base-palette-pomegranate-400";
export const pomegranate500 = "--cnvs-base-palette-pomegranate-500";
export const pomegranate600 = "--cnvs-base-palette-pomegranate-600";
export const fruitPunch100 = "--cnvs-base-palette-fruit-punch-100";
export const fruitPunch200 = "--cnvs-base-palette-fruit-punch-200";
export const fruitPunch300 = "--cnvs-base-palette-fruit-punch-300";
export const fruitPunch400 = "--cnvs-base-palette-fruit-punch-400";
export const fruitPunch500 = "--cnvs-base-palette-fruit-punch-500";
export const fruitPunch600 = "--cnvs-base-palette-fruit-punch-600";
export const rootBeer100 = "--cnvs-base-palette-root-beer-100";
export const rootBeer200 = "--cnvs-base-palette-root-beer-200";
export const rootBeer300 = "--cnvs-base-palette-root-beer-300";
export const rootBeer400 = "--cnvs-base-palette-root-beer-400";
export const rootBeer500 = "--cnvs-base-palette-root-beer-500";
export const rootBeer600 = "--cnvs-base-palette-root-beer-600";
export const toastedMarshmallow100 = "--cnvs-base-palette-toasted-marshmallow-100";
export const toastedMarshmallow200 = "--cnvs-base-palette-toasted-marshmallow-200";
export const toastedMarshmallow300 = "--cnvs-base-palette-toasted-marshmallow-300";
export const toastedMarshmallow400 = "--cnvs-base-palette-toasted-marshmallow-400";
export const toastedMarshmallow500 = "--cnvs-base-palette-toasted-marshmallow-500";
export const toastedMarshmallow600 = "--cnvs-base-palette-toasted-marshmallow-600";
export const licorice100 = "--cnvs-base-palette-licorice-100";
export const licorice200 = "--cnvs-base-palette-licorice-200";
export const licorice300 = "--cnvs-base-palette-licorice-300";
export const licorice400 = "--cnvs-base-palette-licorice-400";
export const licorice500 = "--cnvs-base-palette-licorice-500";
export const licorice600 = "--cnvs-base-palette-licorice-600";
export const soap100 = "--cnvs-base-palette-soap-100";
export const soap200 = "--cnvs-base-palette-soap-200";
export const soap300 = "--cnvs-base-palette-soap-300";
export const soap400 = "--cnvs-base-palette-soap-400";
export const soap500 = "--cnvs-base-palette-soap-500";
export const soap600 = "--cnvs-base-palette-soap-600";
export const frenchVanilla100 = "--cnvs-base-palette-french-vanilla-100";
export const frenchVanilla200 = "--cnvs-base-palette-french-vanilla-200";
export const frenchVanilla300 = "--cnvs-base-palette-french-vanilla-300";
export const frenchVanilla400 = "--cnvs-base-palette-french-vanilla-400";
export const frenchVanilla500 = "--cnvs-base-palette-french-vanilla-500";
export const frenchVanilla600 = "--cnvs-base-palette-french-vanilla-600";
export const blackPepper100 = "--cnvs-base-palette-black-pepper-100";
export const blackPepper200 = "--cnvs-base-palette-black-pepper-200";
export const blackPepper300 = "--cnvs-base-palette-black-pepper-300";
export const blackPepper400 = "--cnvs-base-palette-black-pepper-400";
export const blackPepper500 = "--cnvs-base-palette-black-pepper-500";
export const blackPepper600 = "--cnvs-base-palette-black-pepper-600";
export const coconut100 = "--cnvs-base-palette-coconut-100";
export const coconut200 = "--cnvs-base-palette-coconut-200";
export const coconut300 = "--cnvs-base-palette-coconut-300";
export const coconut400 = "--cnvs-base-palette-coconut-400";
export const coconut500 = "--cnvs-base-palette-coconut-500";
export const coconut600 = "--cnvs-base-palette-coconut-600";
export const cappuccino100 = "--cnvs-base-palette-cappuccino-100";
export const cappuccino200 = "--cnvs-base-palette-cappuccino-200";
export const cappuccino300 = "--cnvs-base-palette-cappuccino-300";
export const cappuccino400 = "--cnvs-base-palette-cappuccino-400";
export const cappuccino500 = "--cnvs-base-palette-cappuccino-500";
export const cappuccino600 = "--cnvs-base-palette-cappuccino-600";
export const size0 = "--cnvs-base-size-0";
export const size25 = "--cnvs-base-size-25";
export const size50 = "--cnvs-base-size-50";
export const size75 = "--cnvs-base-size-75";
export const size100 = "--cnvs-base-size-100";
export const size125 = "--cnvs-base-size-125";
export const size150 = "--cnvs-base-size-150";
export const size175 = "--cnvs-base-size-175";
export const size200 = "--cnvs-base-size-200";
export const size225 = "--cnvs-base-size-225";
export const size250 = "--cnvs-base-size-250";
export const size300 = "--cnvs-base-size-300";
export const size350 = "--cnvs-base-size-350";
export const size400 = "--cnvs-base-size-400";
export const size450 = "--cnvs-base-size-450";
export const size500 = "--cnvs-base-size-500";
export const size600 = "--cnvs-base-size-600";
export const size700 = "--cnvs-base-size-700";
export const size800 = "--cnvs-base-size-800";
export const size900 = "--cnvs-base-size-900";
export const size1000 = "--cnvs-base-size-1000";
export const size1100 = "--cnvs-base-size-1100";
export const size1200 = "--cnvs-base-size-1200";
export const size1300 = "--cnvs-base-size-1300";
export const size1400 = "--cnvs-base-size-1400";
export const fontFamily50 = "--cnvs-base-font-family-50";
export const fontFamily100 = "--cnvs-base-font-family-100";
export const fontFamily200 = "--cnvs-base-font-family-200";
export const fontWeight300 = "--cnvs-base-font-weight-300";
export const fontWeight400 = "--cnvs-base-font-weight-400";
export const fontWeight500 = "--cnvs-base-font-weight-500";
export const fontWeight700 = "--cnvs-base-font-weight-700";
export const letterSpacing50 = "--cnvs-base-letter-spacing-50";
export const letterSpacing100 = "--cnvs-base-letter-spacing-100";
export const letterSpacing150 = "--cnvs-base-letter-spacing-150";
export const letterSpacing200 = "--cnvs-base-letter-spacing-200";
export const baseUnit = "--cnvs-base-unit";
export const extendedDragonFruit100 = "--cnvs-base-extended-palette-dragon-fruit-100";
export const extendedDragonFruit200 = "--cnvs-base-extended-palette-dragon-fruit-200";
export const extendedDragonFruit300 = "--cnvs-base-extended-palette-dragon-fruit-300";
export const extendedDragonFruit400 = "--cnvs-base-extended-palette-dragon-fruit-400";
export const extendedDragonFruit500 = "--cnvs-base-extended-palette-dragon-fruit-500";
export const extendedDragonFruit600 = "--cnvs-base-extended-palette-dragon-fruit-600";
export const fontSize25 = "--cnvs-base-font-size-25";
export const fontSize50 = "--cnvs-base-font-size-50";
export const fontSize75 = "--cnvs-base-font-size-75";
export const fontSize100 = "--cnvs-base-font-size-100";
export const fontSize125 = "--cnvs-base-font-size-125";
export const fontSize150 = "--cnvs-base-font-size-150";
export const fontSize200 = "--cnvs-base-font-size-200";
export const fontSize250 = "--cnvs-base-font-size-250";
export const fontSize300 = "--cnvs-base-font-size-300";
export const fontSize400 = "--cnvs-base-font-size-400";
export const fontSize500 = "--cnvs-base-font-size-500";
export const fontSize600 = "--cnvs-base-font-size-600";
export const fontSize750 = "--cnvs-base-font-size-750";
export const fontSize900 = "--cnvs-base-font-size-900";
export const fontSize1050 = "--cnvs-base-font-size-1050";
export const lineHeight50 = "--cnvs-base-line-height-50";
export const lineHeight100 = "--cnvs-base-line-height-100";
export const lineHeight150 = "--cnvs-base-line-height-150";
export const lineHeight200 = "--cnvs-base-line-height-200";
export const lineHeight250 = "--cnvs-base-line-height-250";
export const lineHeight300 = "--cnvs-base-line-height-300";
export const lineHeight350 = "--cnvs-base-line-height-350";
export const lineHeight400 = "--cnvs-base-line-height-400";
export const lineHeight500 = "--cnvs-base-line-height-500";
export const lineHeight600 = "--cnvs-base-line-height-600";
export const lineHeight750 = "--cnvs-base-line-height-750";
export const lineHeight900 = "--cnvs-base-line-height-900";
export const lineHeight1050 = "--cnvs-base-line-height-1050";
